### DISCUSSION TITLE
I'm trying to contribute, but it says I don't have permission.

### DISCUSSION THREAD

**Discussion by parkgang** [trpc/trpc#3146] [General]
I want to add my project to the Awesome tRPC Collection.

For this, I read the contribution guidelines and tried to create a branch, but the UI created on the github web site was not displayed.

![image](https://user-images.githubusercontent.com/63892989/200811834-19f14d36-2b76-4696-8fb7-f0cb60b8051e.png)

I tried it with CLI just in case, but it is 403. Do you know what the problem is?

![image](https://user-images.githubusercontent.com/63892989/200811988-70f0ba17-b0c2-4a89-8461-186c08fa0fdc.png)

**Comment by KATT** [comment]
You need to fork the repo and push to your own fork

**Reply by parkgang** [reply]
you're right. I just realized I made a mistake and was trying again. Thank you for answer.
